Environment for every PR

waveywaves opened the pull request:
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

mertmit committed:
feat(wip): back-end support for multiple base

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: baseApis

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: meta-diff apis for multiple base

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: front-end allow multiple base per project

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: meta-diff apis for single base

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: multiple source back-end

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: multiple source front-end

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: multiple source project prefix

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: multiple source sync views

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: show only tables within same base as child table for LTAR

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: team & settings modal data sources tab revised

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: missing icon

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: edit base

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: disable base edit/delete if meta base

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: base type checks

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: delete base

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: multiple base handle import & sync

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: use model id instead of title for ui navigation

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: import template editor

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: rename table

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: ui improvements for multiple source interfaces

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: ui improvements for multiple source

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: order and ui control for bases

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: draggable data sources

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: show bases even if no tables are present

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: restrict deleting first base

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: TreeView css

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: refresh data sources on sync metadata

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: hide base if no result for search query

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: improved show/hide base checkbox

Signed-off-by: mertmit <mertmit99@gmail.com>

waveywaves committed:
chore: previews for every PR release using Uffizzi

add github workflows to take the prebuilt docker image for the PR to
create a preview deployment on Uffizzi. Comment will be posted to every
PR on a successful deployment.

mertmit committed:
feat: suffix base alias for duplicate tabs

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: handle exception for metaDiff api

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: allow hiding base zero

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: base reordering logic

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: drop base_id fk restriction for audit table

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: avoid overlapped reload of bases

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: use checkbox instead of icons

Signed-off-by: mertmit <mertmit99@gmail.com>

mustafapc19 committed:
fix(test): Integrated multiple sources changes with playwright

mertmit committed:
fix: update/delete base

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: table reordering

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: change mysql icon

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: TreeView base undefined check

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: make ApiTokenManagement single root to allow prop inheritance

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: multiple-source front end improvements

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: implement latest playwright changes to multiple-source

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: drop duplicate ids

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: apps.vue disable sidebar

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: awakened data sources

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: make base name bold

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: bases panel

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: change data sources show/hide checkbox to switch

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: base show/hide improved handling

Signed-off-by: mertmit <mertmit99@gmail.com>

axisofentropy commented on .github/workflows/uffizzi-build.yml:
```diff
@@ -0,0 +1,161 @@
+        uses: actions/checkout@v3
+      - name: Render Compose File
+        run: |
+          NOCODB_IMAGE=$(echo ${{ needs.build-nocodb.outputs.tags }})
```
Does this need the `echo`? Should it work with just the substitution?

axisofentropy commented on .github/workflows/uffizzi-preview.yml:
```diff
@@ -0,0 +1,88 @@
+    name: Use Remote Workflow to Preview on Uffizzi
+    needs:
+      - cache-compose-file
+    uses: UffizziCloud/preview-action/.github/workflows/reusable.yaml@v2
```
`@v2.6.1`!

axisofentropy commented:
This looks good. Great work integrating with their existing workflow.

I like that they specified `ready_for_review` as an event trigger, and they specified `paths`. We should consider adding these in our examples. https://github.com/waveywaves/nocodb/blob/d5e67e7abd53c9504cdbbbb6b6402ff4c28abc9f/.github/workflows/release-pr.yml#L9-L10

axisofentropy commented on .github/workflows/uffizzi-preview.yml:
```diff
@@ -0,0 +1,88 @@
+      # and this reusable workflow will delete the preview deployment.
+      compose-file-cache-key: ${{ needs.cache-compose-file.outputs.compose-file-cache-key }}
+      compose-file-cache-path: docker-compose.rendered.yml
+      server: https://app.uffizzi.com/
```
Remove trailing `/`.

axisofentropy commented on .github/uffizzi/docker-compose.uffizzi.yml:
```diff
@@ -0,0 +1,24 @@
+      POSTGRES_DB: root_db
+  nocodb:
+    image: "${NOCODB_IMAGE}"
+    ports:
+      - "8080:8080"
+    restart: always
```
I don't think `ports` or `restart` are used by Uffizzi.

waveywaves replied:
keeping them for resusability with docker-compose

axisofentropy commented on .github/workflows/release-pr.yml:
```diff
@@ -94,9 +95,46 @@ jobs:
+  # Create a preview for the pull request
+  preview-pull-request:
+    name: "Trigger Uffizzi Preview"
+    if: ${{ github.event.pull_request.head.repo.full_name == github.repository && github.actor != 'dependabot[bot]' && github.event.pull_request.draft == false && github.base_ref == 'develop' && github.event.action != 'closed' }}
```
I like these conditions and we should probably make most of them standard.

waveywaves replied:
I agree

axisofentropy commented on .github/workflows/release-pr.yml:
```diff
@@ -94,9 +95,46 @@ jobs:
+        run:  |
+          cat << EOF > event.json
+          ${{ toJSON(github.event) }} 
+          
```
We don't want this empty line in the JSON file.

waveywaves replied:
removing it

mertmit committed:
fix: cellIcon for multiple source

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: change treeview root to bases if more than one base

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: shared view sqlUis

Signed-off-by: mertmit <mertmit99@gmail.com>

pranavxc committed:
feat(gui): allow pasting value directly to cell

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
feat: convert type when copy paste data within grid

Signed-off-by: Pranav C <pranavxc@gmail.com>

mustafapc19 committed:
feat(test): Added test for clipboard support(disabled for now)

pranavxc committed:
fix: avoid opening dropdown on cmd/alt/shift/ctrl key press

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
fix: sync data after copying

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
fix: update value without making editable

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
feat(gui): show warning if paste not supported in cell

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
feat(gui): copying belongs to column

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
fix(gui): handle invalid data when copying belongs to cell data

Signed-off-by: Pranav C <pranavxc@gmail.com>

mertmit committed:
fix: PR requested changes

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: auto-expand bases in TreeView

Signed-off-by: mertmit <mertmit99@gmail.com>

pranavxc committed:
fix(gui): validate attachment value

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
chore(gui): lint

Signed-off-by: Pranav C <pranavxc@gmail.com>

mustafapc19 committed:
fix(playwright): Enabled grid operation test

pranavxc committed:
refactor(gui): group import and cleanup

Signed-off-by: Pranav C <pranavxc@gmail.com>

mertmit committed:
feat: sqlite alter table change column

Signed-off-by: mertmit <mertmit99@gmail.com>

pranavxc committed:
fix(gui): avoiding blocking click event propagation

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
fix(gui): add validation for paste data

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
fix(gui): convert checkbox value to true/false while copying

Signed-off-by: Pranav C <pranavxc@gmail.com>

mertmit committed:
fix: primary key operations for sqlite

Signed-off-by: mertmit <mertmit99@gmail.com>

pranavxc committed:
fix(gui): toast message update

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
test(playwright): test for column duplicate

Signed-off-by: Pranav C <pranavxc@gmail.com>

o1lab committed:
New translations en.json (Turkish)

o1lab committed:
New translations en.json (Slovenian)

o1lab committed:
New translations en.json (Spanish)

o1lab committed:
New translations en.json (Russian)

o1lab committed:
New translations en.json (Portuguese)

o1lab committed:
New translations en.json (Polish)

o1lab committed:
New translations en.json (Norwegian)

o1lab committed:
New translations en.json (Dutch)

o1lab committed:
New translations en.json (Korean)

o1lab committed:
New translations en.json (Japanese)

o1lab committed:
New translations en.json (Italian)

o1lab committed:
New translations en.json (Hebrew)

o1lab committed:
New translations en.json (Finnish)

o1lab committed:
New translations en.json (German)

o1lab committed:
New translations en.json (Danish)

o1lab committed:
New translations en.json (Arabic)

o1lab committed:
New translations en.json (Bengali, India)

mertmit committed:
fix: handle foreign key and view restrictions

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix(test): drop view restricting column edit for sqlite

Signed-off-by: mertmit <mertmit99@gmail.com>

dstala committed:
Merge pull request #4548 from nocodb/l10n_develop_2

New Crowdin updates

mertmit committed:
feat: tree-shaking lodash in sqlClients

Signed-off-by: mertmit <mertmit99@gmail.com>

dstala committed:
Merge pull request #4482 from nocodb/feat/keyboard-manoeuvre

feat: keyboard manoeuvre

dstala committed:
Merge pull request #4478 from nocodb/docs/keyboard

docs: keyboard maneuver

pranavxc committed:
test(playwright): checkbox toggle correction

Signed-off-by: Pranav C <pranavxc@gmail.com>

mustafapc19 committed:
feat(playwright): Fixed issue with stress test

pranavxc committed:
chore: lint

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
fix(gui): year validation correction

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
chore(gui): lint

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
fix(gui): allow rating cell copy even in editable state

Signed-off-by: Pranav C <pranavxc@gmail.com>

dstala committed:
Merge pull request #4514 from nocodb/feat/copy-paste-cell

Feat: Grid - copy paste cell data

dstala committed:
Merge pull request #2780 from nocodb/dependabot/npm_and_yarn/packages/nocodb/terser-4.8.1

chore(deps): bump terser from 4.8.0 to 4.8.1 in /packages/nocodb

pranavxc committed:
test(playwright): add test for insert before/after option

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
test(playwright): add test for hide field option

Signed-off-by: Pranav C <pranavxc@gmail.com>

wingkwong committed:
Merge pull request #3668 from nocodb/feat/sqlite-change-column

feat: sqlite alter table change column

pranavxc committed:
test(playwright): test for add sort options in column menu

Signed-off-by: Pranav C <pranavxc@gmail.com>

wingkwong committed:
chore(nc-gui): add active: modalVisible to QRcode

wingkwong committed:
chore(nc-gui): lint

wingkwong committed:
chore(nocodb): lint

mustafapc19 committed:
feat(playwright): Added tests for single select, multi select, single line text, long text and attachments

mustafapc19 committed:
feat(playwright): Added tests for Phone number

pranavxc committed:
chore(github-action): add test folder in ci-cd trigger filter

Signed-off-by: Pranav C <pranavxc@gmail.com>

mustafapc19 committed:
feat(playwright): Added tests for Email, URL, Decimal and Percent

pranavxc committed:
chore(gui): move label to i18n

Signed-off-by: Pranav C <pranavxc@gmail.com>

wingkwong committed:
feat(nc-gui): add $e for shortcuts

wingkwong committed:
fix(nc-gui): change $e type from a -> c

pranavxc committed:
test(playwright): corrections

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
test(playwright): type correction

Url => URL

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
Merge pull request #4547 from nocodb/feat/field-option-followup

test: Field menu followup - playwright test

wingkwong committed:
Merge pull request #4550 from nocodb/feat/shortcut-e

feat(nc-gui): add $e for shortcuts

pranavxc committed:
fix(gui): add rating validation while pasting

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
Merge pull request #4551 from nocodb/fix/add-rating-validation

fix(gui): Grid - add rating validation while pasting

github-actions[bot] committed:
Merge pull request #4554 from nocodb/develop

wingkwong committed:
[create-pull-request] automated change

Signed-off-by: GitHub <noreply@github.com>

github-actions[bot] committed:
Merge pull request #4555 from nocodb/release/0.100.0

pranavxc committed:
fix(gui): avoid clearing cell data if it's a non-typable column

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
chore: update sdk path

Signed-off-by: Pranav C <pranavxc@gmail.com>

mustafapc19 committed:
feat(playwright): Added tests for Currency, Duration, Rating

pranavxc committed:
fix: pass argument in correct order

Signed-off-by: Pranav C <pranavxc@gmail.com>

wingkwong committed:
chore(nc-gui): lint

pranavxc committed:
fix: verify property exist in row object

Signed-off-by: Pranav C <pranavxc@gmail.com>

pranavxc committed:
Merge pull request #4559 from nocodb/fix/4557-cell-data-sync

fix: Grid - Sync unsaved data properly when switching tab

github-actions[bot] committed:
Merge pull request #4560 from nocodb/develop

pranavxc committed:
[create-pull-request] automated change

Signed-off-by: GitHub <noreply@github.com>

github-actions[bot] committed:
Merge pull request #4561 from nocodb/release/0.100.1

pranavxc committed:
chore: lint

Signed-off-by: Pranav C <pranavxc@gmail.com>

dstala committed:
Merge pull request #4556 from nocodb/fix/4493-avoid-clearing-data

fix(gui): avoid clearing cell data if it's a non-typable column

dependabot[bot] committed:
chore(deps): bump socket.io-parser in /packages/nocodb

Bumps [socket.io-parser](https://github.com/socketio/socket.io-parser) from 4.0.4 to 4.0.5.
- [Release notes](https://github.com/socketio/socket.io-parser/releases)
- [Changelog](https://github.com/socketio/socket.io-parser/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io-parser/compare/4.0.4...4.0.5)

---
updated-dependencies:
- dependency-name: socket.io-parser
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

pranavxc committed:
chore: update sdk path

Signed-off-by: Pranav C <pranavxc@gmail.com>

o1lab committed:
New translations en.json (French)

o1lab committed:
New translations en.json (Swedish)

o1lab committed:
New translations en.json (Hindi)

o1lab committed:
New translations en.json (Latvian)

o1lab committed:
New translations en.json (Croatian)

o1lab committed:
New translations en.json (Thai)

o1lab committed:
New translations en.json (Persian)

o1lab committed:
New translations en.json (Indonesian)

o1lab committed:
New translations en.json (Portuguese, Brazilian)

o1lab committed:
New translations en.json (Vietnamese)

o1lab committed:
New translations en.json (English)

o1lab committed:
New translations en.json (Chinese Traditional)

o1lab committed:
New translations en.json (Chinese Simplified)

o1lab committed:
New translations en.json (Ukrainian)

o1lab committed:
New translations en.json (Turkish)

o1lab committed:
New translations en.json (Slovenian)

o1lab committed:
New translations en.json (Spanish)

o1lab committed:
New translations en.json (Russian)

o1lab committed:
New translations en.json (Portuguese)

o1lab committed:
New translations en.json (Polish)

o1lab committed:
New translations en.json (Norwegian)

o1lab committed:
New translations en.json (Dutch)

o1lab committed:
New translations en.json (Korean)

o1lab committed:
New translations en.json (Japanese)

o1lab committed:
New translations en.json (Italian)

o1lab committed:
New translations en.json (Hebrew)

o1lab committed:
New translations en.json (Finnish)

o1lab committed:
New translations en.json (German)

o1lab committed:
New translations en.json (Danish)

o1lab committed:
New translations en.json (Arabic)

o1lab committed:
New translations en.json (Bengali, India)

o1lab committed:
Merge pull request #4562 from nocodb/l10n_develop_2

New Crowdin updates

dstala committed:
Merge pull request #4350 from nocodb/dependabot/npm_and_yarn/packages/nocodb/socket.io-parser-4.0.5

chore(deps): bump socket.io-parser from 4.0.4 to 4.0.5 in /packages/nocodb

dependabot[bot] committed:
chore(deps): bump decode-uri-component in /packages/nc-plugin

Bumps [decode-uri-component](https://github.com/SamVerschueren/decode-uri-component) from 0.2.0 to 0.2.2.
- [Release notes](https://github.com/SamVerschueren/decode-uri-component/releases)
- [Commits](https://github.com/SamVerschueren/decode-uri-component/compare/v0.2.0...v0.2.2)

---
updated-dependencies:
- dependency-name: decode-uri-component
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump decode-uri-component in /packages/noco-docs

Bumps [decode-uri-component](https://github.com/SamVerschueren/decode-uri-component) from 0.2.0 to 0.2.2.
- [Release notes](https://github.com/SamVerschueren/decode-uri-component/releases)
- [Commits](https://github.com/SamVerschueren/decode-uri-component/compare/v0.2.0...v0.2.2)

---
updated-dependencies:
- dependency-name: decode-uri-component
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump decode-uri-component in /packages/nc-cli

Bumps [decode-uri-component](https://github.com/SamVerschueren/decode-uri-component) from 0.2.0 to 0.2.2.
- [Release notes](https://github.com/SamVerschueren/decode-uri-component/releases)
- [Commits](https://github.com/SamVerschueren/decode-uri-component/compare/v0.2.0...v0.2.2)

---
updated-dependencies:
- dependency-name: decode-uri-component
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump @xmldom/xmldom from 0.7.6 to 0.7.9 in /packages/nc-cli

Bumps [@xmldom/xmldom](https://github.com/xmldom/xmldom) from 0.7.6 to 0.7.9.
- [Release notes](https://github.com/xmldom/xmldom/releases)
- [Changelog](https://github.com/xmldom/xmldom/blob/master/CHANGELOG.md)
- [Commits](https://github.com/xmldom/xmldom/compare/0.7.6...0.7.9)

---
updated-dependencies:
- dependency-name: "@xmldom/xmldom"
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump decode-uri-component in /packages/nocodb

Bumps [decode-uri-component](https://github.com/SamVerschueren/decode-uri-component) from 0.2.0 to 0.2.2.
- [Release notes](https://github.com/SamVerschueren/decode-uri-component/releases)
- [Commits](https://github.com/SamVerschueren/decode-uri-component/compare/v0.2.0...v0.2.2)

---
updated-dependencies:
- dependency-name: decode-uri-component
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump decode-uri-component from 0.2.0 to 0.2.2

Bumps [decode-uri-component](https://github.com/SamVerschueren/decode-uri-component) from 0.2.0 to 0.2.2.
- [Release notes](https://github.com/SamVerschueren/decode-uri-component/releases)
- [Commits](https://github.com/SamVerschueren/decode-uri-component/compare/v0.2.0...v0.2.2)

---
updated-dependencies:
- dependency-name: decode-uri-component
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

wingkwong committed:
Merge pull request #4569 from nocodb/dependabot/npm_and_yarn/decode-uri-component-0.2.2

chore(deps): bump decode-uri-component from 0.2.0 to 0.2.2

wingkwong committed:
Merge pull request #4568 from nocodb/dependabot/npm_and_yarn/packages/nocodb/decode-uri-component-0.2.2

chore(deps): bump decode-uri-component from 0.2.0 to 0.2.2 in /packages/nocodb

wingkwong committed:
Merge pull request #4567 from nocodb/dependabot/npm_and_yarn/packages/nc-cli/xmldom/xmldom-0.7.9

chore(deps): bump @xmldom/xmldom from 0.7.6 to 0.7.9 in /packages/nc-cli

wingkwong committed:
Merge pull request #4566 from nocodb/dependabot/npm_and_yarn/packages/nc-cli/decode-uri-component-0.2.2

chore(deps): bump decode-uri-component from 0.2.0 to 0.2.2 in /packages/nc-cli

wingkwong committed:
Merge pull request #4565 from nocodb/dependabot/npm_and_yarn/packages/noco-docs/decode-uri-component-0.2.2

chore(deps): bump decode-uri-component from 0.2.0 to 0.2.2 in /packages/noco-docs

wingkwong committed:
Merge pull request #4564 from nocodb/dependabot/npm_and_yarn/packages/nc-plugin/decode-uri-component-0.2.2

chore(deps): bump decode-uri-component from 0.2.0 to 0.2.2 in /packages/nc-plugin

pranavxc committed:
refactor: remove outline of search input in single select

Signed-off-by: Pranav C <pranavxc@gmail.com>

wingkwong committed:
feat(nocodb): include error message for resetting password

wingkwong committed:
feat(nocodb): throw e.response.data.msg if possible

wingkwong committed:
fix(nc-gui): type issues

dstala committed:
Merge pull request #4572 from nocodb/refctor/remove-input-outline

Remove outline of search input in single select

wingkwong committed:
Merge pull request #4571 from nocodb/feat/reset-password-error-msg

feat(nocodb): include error message for resetting password

wingkwong committed:
fix(nc-gui): remove incorrect click event and update icon

wingkwong committed:
feat(nc-gui): add isDrawerExist

wingkwong committed:
fix(nc-gui): prevent focus when there is active drawer

pranavxc committed:
Merge pull request #4576 from nocodb/fix/LTAR-focus

fix(nc-gui): unexpected focus on ListItem

github-actions[bot] committed:
Merge pull request #4577 from nocodb/develop

pranavxc committed:
[create-pull-request] automated change

Signed-off-by: GitHub <noreply@github.com>

github-actions[bot] committed:
Merge pull request #4578 from nocodb/release/0.100.2

wingkwong committed:
chore: update sdk local path

wingkwong committed:
fix(nocodb): provide header in XLSX.utils.json_to_sheet

spaudanjo committed:
QrCode: also allow decimals and numbers as referenced values

spaudanjo committed:
QrCode: fix playwright test (make text based selector unambiguous)

mertmit committed:
fix: loadBases if not loaded before handling state in DataSources

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: TreeView scroll

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: move base operations dropdown to root level

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: expand all bases while searching

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
feat: add share base button to top right

Signed-off-by: mertmit <mertmit99@gmail.com>

wingkwong committed:
chore(nc-gui): resolve PR comments

wingkwong committed:
fix(nc-gui): formatTitle type

mertmit committed:
fix: add missing runtime config for appInfo

Signed-off-by: mertmit <mertmit99@gmail.com>

wingkwong committed:
Merge pull request #4574 from nocodb/fix/types

fix(nc-gui): type issues

wingkwong committed:
feat(playwrights): add download xlsx test

wingkwong committed:
feat(playwright): clean up export tests

o1lab committed:
New translations en.json (French)

spaudanjo committed:
Merge branch 'develop' into enable-decimals-and-numbers-for-qr-code

spaudanjo committed:
set playwright version again to fixed one

wingkwong committed:
fix(playwright): use txt instead of csv for fs reader

wingkwong committed:
fix(playwright): CRLF -> LF

wingkwong committed:
fix(playwright): change to UTF-8

wingkwong committed:
fix(playwright): export test

o1lab committed:
Merge pull request #3573 from nocodb/feat/multiple-source

feat: allow project to connect to multiple data sources

o1lab committed:
Merge pull request #4596 from nocodb/l10n_develop_2

New Crowdin updates

wingkwong committed:
fix(playwright): remove \r

dstala committed:
Merge pull request #4585 from humannocode/enable-decimals-and-numbers-for-qr-code

QrCode: also allow decimals and numbers as referenced values

wingkwong committed:
fix(nc-gui): retrieve the correct column name based on the formula

wingkwong committed:
fix(nc-gui): handle new formula case

wingkwong committed:
Merge pull request #4581 from nocodb/fix/export-excel-col-order

fix(nocodb): provide header in XLSX.utils.json_to_sheet

dstala committed:
Merge pull request #4598 from nocodb/fix/formula-raw

fix(nc-gui): retrieve the correct column name based on the formula

wingkwong committed:
fix(nocodb-sdk): include node.value for literal case

wingkwong committed:
chore: update package-lock.json

wingkwong committed:
chore(nc-gui): lint

dstala committed:
Merge pull request #4599 from nocodb/fix/formula-column-with-space

fix(nocodb-sdk): include node.value for literal case

wingkwong committed:
fix(playwright): lint blocking

dstala committed:
Merge pull request #4552 from nocodb/feat/copy-paste-cell

Copy paste cell more tests added

wingkwong committed:
Merge pull request #4460 from waveywaves/uffizzi

chore: previews for every PR release using Uffizzi

waveywaves committed:
Add other DBs to uffizzi preview

waveywaves committed:
trigger preview

mertmit committed:
fix: typescript compile error

Signed-off-by: mertmit <mertmit99@gmail.com>

mertmit committed:
fix: add useNullAsDefault flag for sqlite on root level

Signed-off-by: mertmit <mertmit99@gmail.com>

wingkwong committed:
fix(nc-gui): parse string date if date is not null

wingkwong committed:
Merge pull request #4609 from nocodb/fix/sqlite-warning

fix: add useNullAsDefault flag for sqlite on root level

wingkwong committed:
Merge pull request #4605 from waveywaves/uffizzi-dbs

Add other DBs to uffizzi preview

wingkwong committed:
Merge branch 'develop' into fix/import-csv-date

wingkwong committed:
Merge pull request #4611 from nocodb/fix/import-csv-date

fix(nc-gui): parse string date if date is not null
